package.json

```diff
@@ -44,7 +44,7 @@
     "gulp-jshint": "2.0.0",
     "gulp-rename": "1.2.2",
     "gulp-replace": "0.5.4",
-    "gulp-zip": "3.0.2",
+    "gulp-zip": "3.2.0",
```

Comment:
Whoops, I swear I unstaged that line before committing.  Thanks for catching it, I'll fix it,